android/app-example/src/main/java/com/badoo/ribs/example/rib/foo_bar/FooBar.kt

```diff
@@ -0,0 +1,25 @@
+package com.badoo.ribs.example.rib.foo_bar
+
+import com.badoo.common.rib.ViewFactory
+import com.badoo.common.rib.directory.Directory
+import com.badoo.common.rib.directory.inflateOnDemand
+import com.badoo.ribs.example.R
+import io.reactivex.ObservableSource
+import io.reactivex.functions.Consumer
+
+interface FooBar {
+
+    interface Dependency {
+        fun foobarInput(): ObservableSource<Input>
+        fun foobarOutput(): Consumer<Output>
+        fun ribCustomisation(): Directory
```

Comment:
If we will not rethink customization way, the naming can be improved. 
Directory is not clear, maybe too abstract.

Reply:
Agreed. Probably we can also just roll with using existing Map interface, what do you think?